curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646b07faa74eb103dd52d206.md

```diff
@@ -0,0 +1,109 @@
+---
+id: 646b07faa74eb103dd52d206
+title: Step 14
+challengeType: 0
+dashedName: step-14
+---
+
+# --description--
+
+Target the class named `divider` and do the following:
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed